.github/workflows/ci.yml

```diff
@@ -63,6 +62,7 @@ jobs:
       matrix:
         rust:
           - stable
+          - 1.60.0
```

Comment:
Note that we weren't actually running this before

Reply:
Yeah, we should have. I think the gha generally need some more attention. But that's out of scope